frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/error-components/resource-load-errors.tsx

```diff
@@ -10,6 +10,32 @@ import { K8sKind } from '@console/internal/module/k8s';
 import { TemplateModel } from '@console/internal/models';
 import { DataVolumeModel, VirtualMachineModel } from '../../../models';
 import { getExtraWSQueries } from '../selectors/selectors';
+import { iGetUserTemplateName, iGetUserTemplateValidity } from '../selectors/immutable/template';
+
+const userTemplateError = (state: any, wizardReduxID: string): Error => {
```

Comment:
if we load one use template in that case we don't need a special handler like this one but a generic handler like VMWizardProps.userTemplates have right now